middleware.ts

```
@@ -6,23 +6,17 @@ import { NextResponse, type NextRequest } from "next/server";
-export async function middleware(request: NextRequest) {
-  const requestId = crypto.randomUUID();
-  request.headers.set("x-request-id", requestId);
 
-  let response = NextResponse.next({
-    request,
-  });
+export async function middleware(request: NextRequest) {
+  const response = NextResponse.next();
 
   const supabase = createServerClient(
     process.env.NEXT_PUBLIC_SUPABASE_URL!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)